### PR TITLE
Game modal style improvements

### PR DESCRIPTION
**Pull request by Vasso**

### CODE REVIEW

File: client/static/app/modules/play/controllers/play-join.controller.js [hunk 1]
```diff
@@ -23,17 +24,30 @@ define(function(require) {
...
     dgPlayService.getById(playId).then(function(data) {
+      console.log(data);
       vm.playData = data;
+      setPlayDateFormat();
       setIsPlayerJoined();
       $rootScope.$emit('dg:globalLoader:hide');
     });
 
+    function setPlayDateFormat() {
+      console.log(vm.playData.when)
```
Comment (artyomtrityak) [inline client/static/app/modules/play/controllers/play-join.controller.js]: remove it

File: client/static/app/modules/play/controllers/play-join.controller.js [hunk 2]
```diff
@@ -23,17 +24,30 @@ define(function(require) {
...
+      if (!vm.playData.when) {
+        return;
+      }
+      var dateTime = moment(vm.playData.when).toDate()
+      vm.playData.when = moment(dateTime).format('DD/MM/YYYY hh:mm')
+      // return vm.when = '2000000'
```
Comment (artyomtrityak) [inline client/static/app/modules/play/controllers/play-join.controller.js]: remove it

File: client/static/app/modules/play/controllers/play-join.controller.js [hunk 3]
```diff
@@ -23,17 +24,30 @@ define(function(require) {
...
     vm.state = {
       isLoading: false,
-      isDetailsOpen: false,
+      isDetailsOpen: true,
       alreadyJoined: false,
       limitReached: false
     };
 
     dgPlayService.getById(playId).then(function(data) {
+      console.log(data);
```
Comment (artyomtrityak) [inline client/static/app/modules/play/controllers/play-join.controller.js]: remove

File: client/static/app/modules/play/templates/play-join.tpl.html [hunk 4]
```diff
@@ -6,24 +6,38 @@ <h3 class="m-t-none m-b">Join Game</h3>
     <form role="form" name="joinIns.join_game_form" novalidate>
         <div class="row">
             <div class="col-md-4">
-                <div class="row" style="margin-top: 12px;">
-                    <img class="img-thumbnail img-responsive" ng-src="{{joinIns.playData.img}}" src="http://placehold.it/290x160">
+                <div class=" text-center">
```
Comment (artyomtrityak) [inline client/static/app/modules/play/templates/play-join.tpl.html]: remove space

File: client/static/app/modules/play/templates/play-new.tpl.html [hunk 5]
```diff
@@ -3,95 +3,96 @@
     <h3 class="m-t-none m-b">New Play</h3>
     <p>This is create new play description</p>
 
-    <form role="form" name="playNewIns.create_game_form" novalidate 
+    <form role="form" name="playNewIns.create_game_form" novalidate
         dg-loading-locker is-loading="playNewIns.state.isLoading">
         <div class="row">
             <div class="col-md-4">
-                <div class="row" style="margin-top: 12px;">
```
Comment (artyomtrityak) [inline client/static/app/modules/play/templates/play-new.tpl.html]: remove inline styles

File: client/static/assets/scss/plays.scss [hunk 6]
```diff
@@ -0,0 +1,20 @@
+.players-count-form {
```
Comment (artyomtrityak) [inline client/static/assets/scss/plays.scss]: BEM?

**Comment by artyomtrityak** [comment]
make changes / answers and merge

File: client/static/assets/scss/plays.scss (hunk 6 above)
Comment (Vasso) [inline client/static/assets/scss/plays.scss]: will create custom classes for block elements

**Comment by Vasso** [comment]
updated

**Comment by artyomtrityak** [comment]
looks good, merge it

**Pull request merged**